### DISCUSSION TITLE
Signalement anomalie patch 1.1.0

### DISCUSSION THREAD

**Discussion by la-symeyronie** [L-Sherry/French-CC#11] [Q&A]
Bonjour,
J'ai remarqué une anomalie sur le dernier patch : la traduction des quêtes exclusives aux consoles sont recherché sur trois fichiers que vous avez, à priori, oublié de mettre en ligne. La dernière mise à jour est, du coup, incomplète.
J'en profite pour vous remercier pour votre travail, permettant aux personnes ayant des difficultés avec l'anglais de découvrir ce jeux.
En espérant que ma remarque vous aura été utile.

**Comment by L-Sherry** [comment]
Bonjour,

C'est corrigé. Désolé pour l'erreur. Je vais sortir 1.2.0 du coup.

Merci !
